src/Path/Term.hs

```diff
@@ -61,5 +61,5 @@ prettyTermInContext
 prettyTermInContext alg = go
   where go :: forall n . Vec n Doc -> Term sig (Var (Fin n) a) -> Prec
         go ctx = \case
-          Var v -> atom (var (pretty . (ctx !)) pretty v)
```

Comment:
`prettyprinter`’s `Doc` type doesn’t have a `Pretty` instance, so I was prompted to remove a bunch of cases where we were using `pretty` on `Doc`. While our `Doc` type _does_ have a `Pretty` instance, I’ve removed a bunch of these redundant uses of `pretty` anyway for clarity.